app/constants/__init__.py

```diff
@@ -99,3 +99,7 @@
 }
 
 OPENRECORDS_DL_EMAIL = "openrecords@records.nyc.gov"
+
```

Comment:
Constant has now been removed.

New type is not needed. The duplicate denial reasons have been removed for reason type `closing`. `request.forms.py` has been updated accordingly in respect to this change.